client/client.go

```diff
@@ -131,6 +133,35 @@ func (c *client) Start(ctx context.Context, execPath string, opts ...OpOption) (
 	return c.controlc.Start(ctx, req)
 }
 
+func (c *client) DeployBlockchains(ctx context.Context, opts ...OpOption) (*rpcpb.DeployBlockchainsResponse, error) {
+	ret := &Op{}
+	ret.applyOpts(opts)
+
+	req := &rpcpb.DeployBlockchainsRequest{}
+	if ret.whitelistedSubnets != "" {
+		req.WhitelistedSubnets = &ret.whitelistedSubnets
+	}
+	if ret.pluginDir != "" {
+		req.PluginDir = &ret.pluginDir
+	}
+	if len(ret.blockchainSpecs) > 0 {
```

Comment:
At this moment, it does nothing. Changed so as the server returns error if no blockchain is specified. Also remove the pluginDir parameter. It will use the network level plugin parameter that was set either with Start() or LoadSnapshot().